package.json

```diff
@@ -196,6 +197,7 @@
     "mobx-observable-history": "^1.0.3",
     "mock-fs": "^4.12.0",
     "node-machine-id": "^1.1.12",
+    "node-notifier": "^9.0.0",
```

Comment:
Where is this used?

Reply:
Sorry, thought I removed all the packages I added while iterating. Will remove